lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/document_builder_api_common/document_builder.rb

```diff
@@ -46,6 +50,12 @@ def create_xlsx_from_sample_data
       path_to_downloaded_xlsx_file
     end
 
+    def input_name_company_position(name = 'Ivan Petrov', company = 'Heartwell', position = 'QA Engineer')
```

Comment:
beter use `params = {}` as parameter
and call it by `params.fetch(:name, 'Ivan Petrov')`